### ISSUE TITLE
non-makepkg-build staging with NTSYNC on Fedora 41: proton-tkg-additions.patch fails to apply

### ISSUE THREAD

**Issue by AllenTzul** [Frogging-Family/wine-tkg-git#1326]
Based on wine-staging **51ccd95c49c** Release **9.22**

```
proton-tkg-additions.patch -- ( Using additional Proton-tkg patches )
patching file dlls/winex11.drv/window.c
Hunk #1 FAILED at 2615.
Hunk #2 FAILED at 2649.
2 out of 2 hunks FAILED -- saving rejects to file dlls/winex11.drv/window.c.rej
patching file dlls/winex11.drv/event.c
Hunk #1 FAILED at 1411.
1 out of 1 hunk FAILED -- saving rejects to file dlls/winex11.drv/event.c.rej
```

**Comment by Kerrung** [comment]
Hi! I have the same issue:
[last_build_config.log](https://github.com/user-attachments/files/17878891/last_build_config.log)
[prepare.log](https://github.com/user-attachments/files/17878892/prepare.log)
![Screenshot From 2024-11-23 12-21-30](https://github.com/user-attachments/assets/587d21e6-3afb-43fa-92ca-1b0edda88255)

**Comment by rankynbass** [comment]
Two parts of this patchset no longer apply at all. Work done in 9.22 on winex11.drv has completely changed the code being patched: [26f3edd](https://github.com/wine-mirror/wine/commit/26f3edd4531a85bf2d873a34eaf3514020bf2ea4) and [0f1d999](https://github.com/wine-mirror/wine/commit/0f1d999bb878ea54214057d3662c116271ec4812).

These two sections need to be removed for the patch to apply:
* winex11.drv: Allow the application to change window size and states during PropertyNotify.
* winex11.drv: Call SetForegroundWindow instead of SetActiveWindow on restore.

**Comment by AllenTzul** [comment]
Fixed by 7db90c1. Thank you.
